apps/mitt-konto/src/MittKonto/Main/Views/UserView/Subscription/Elements.purs

```diff
@@ -275,11 +275,11 @@ subscriptionUpdates self@{ props: props@{ subscription: sub@{ subsno, package }
                           self.setState _
                             { wrapperProgress = AsyncWrapper.Success $ Just "Tillfällig adressändring har tagits bort",
                             pendingAddressChanges = toMaybe newSubscription.pendingAddressChanges }
-                          Tracking.deleteTempAddressChange (show props.subscription.cusno) (show props.subscription.subsno) startDate' endDate' "success"
+                          Tracking.deleteTempAddressChange props.subscription.cusno (show props.subscription.subsno) startDate' endDate' "success"
```

Comment:
I think we should add the `Subsno` type as well, but we can do it in a different PR.

Reply:
I would leave it for a later one. I did this for `Cusno`s now since I needed to do something to get to compare `Cusno`s in subscriptions and in user data.